CONTRIBUTING.md

```diff
@@ -1,17 +1,27 @@
 <!-- Also update README.md (duplicate) -->
+## How can I contribute?
 
-# How can I contribute?
-
-1. 💻 Follow the [Getting started](./dev_docs/01_getting_started.md) to set up your development server. At the bottom of that page, you will also find links that will take you to guidelines for the most common tasks.
-2. 🔍 Search for issues tagged as [help wanted](https://github.com/learningequality/kolibri-design-system/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22+no%3Aassignee+) or [good first issue](https://github.com/learningequality/kolibri-design-system/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22+no%3Aassignee).
-3. 🗣️ Ask us for an assignment in the comments of an issue you've chosen.
-
-Note that in times of increased contributions activity, it may take us a few days to reply. If you don't hear from us within a week, please reach out via [Kolibri Design System GitHub Discussions](https://github.com/learningequality/kolibri-design-system/discussions).
+1. 💻 **Follow the [Getting started](./dev_docs/01_getting_started.md) to set up your development server.** At the bottom of the page, you can also find links that will take you to guidelines for the most common tasks.
+2. 🔍 **Search for issues tagged as [help wanted](https://github.com/learningequality/kolibri-design-system/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22+no%3Aassignee+) or [good first issue](https://github.com/learningequality/kolibri-design-system/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22+no%3Aassignee).**
+4. 🗣️ **Ask us for an assignment in the comments of an issue you've chosen.** Please request assignment of a reasonable amount of issues at a time. Once you finish your current issue or two, you are welcome to ask for more.
 
 **❓ Where to ask questions**
 
-- For anything development related, refer to the [developer documentation](./dev_docs/) at first. Some answers may already be there.
+- For anything development related, refer to the [Developer documentation](./dev_docs/) at first. Some answers may already be there.
 - For questions related to a specific issue or assignment requests, use the corresponding issue's comments section.
-- Visit [Kolibri Design System GitHub Discussions](https://github.com/learningequality/kolibri-design-system/discussions) to ask about anything related to contributing to Kolibri Design System, troubleshoot development server issues, or connect with other contributors.
+- Visit [GitHub Discussions](https://github.com/learningequality/kolibri-design-system/discussions) to ask about anything related to contributing or to troubleshoot development server issues.
```

Comment:
**non-blocking:** i think it might be helpful to either return to the previous wording for the link above or alter it somewhat, as i believe it is a little confusing that this link and the ones on line 16/21 have the same name, "Github Discussions," but one leads to the KDS discussion board while the others lead to the discussion board for Kolibri. 
 
i definitely see the utility in sharing links to both, but think they should have differentiated names to add clarity for users viewing the rich text version

Reply:
> i think it might be helpful to either return to the previous wording for the link above or alter it somewhat, as i believe it is a little confusing that this link and the ones on line 16/21 have the same name, "Github Discussions," but one leads to the KDS discussion board while the others lead to the discussion board for Kolibri.

That's a mistake. The link on line 16 shouldn't point to Kolibri, but to the Design System too, so it will be the same like other links in this document. I will fix it. Thank you, good catch.